src/interface/PubsubEvent.ts

```diff
@@ -15,10 +16,19 @@ export abstract class PubsubEvent<T extends Record<string, any> | Buffer> {
     // Get publish options.
     getOptions = (): PublishOptions => this.options;
 
+    // Get local publishing option.
+    localEventEnabled = (): boolean => this.fireLocally;
```

Comment:
what if need both, local events and RabbitMQ?

Reply:
As I see, `fireLocally` is `true` by default

Reply:
@ddimitrioglo yes, it fires local by default, then if event is instance of PubSubEvent, it fires the rabbitmq, but for cases when you want only pubsub, you set it to false